source/aggregation.txt

```diff
@@ -0,0 +1,197 @@
+   database from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+   free MongoDB Atlas cluster and load the sample datasets, see the :atlas:`Get Started with Atlas
+   </getting-started>` guide.
+
+To perform an aggregation, pass an array containing the aggregation pipeline
+stages to the ``MongoDB\Collection::aggregate()`` method.
+
+The following code example produces a count of the number of bakeries in each borough
+of New York. To do so, it uses an aggregation pipeline that contains the following stages:
+
+- :manual:`$match </reference/operator/aggregation/match/>` stage to filter for documents
+  in which the ``cuisine`` field contains the value ``'Bakery'``
+
+- :manual:`$group </reference/operator/aggregation/group/>` stage to group the matching
+  documents by the ``borough`` field, accumulating a count of documents for each distinct
+  value
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/aggregation.php
+      :start-after: start-match-group
+      :end-before: end-match-group
+      :language: php
+      :dedent:
+
+   .. output:: 
+      :visible: false
+      
+      {"_id":"Brooklyn","count":173}
+      {"_id":"Queens","count":204}
+      {"_id":"Bronx","count":71}
+      {"_id":"Staten Island","count":20}
+      {"_id":"Missing","count":2}
+      {"_id":"Manhattan","count":221}
+
+Explain an Aggregation
+~~~~~~~~~~~~~~~~~~~~~~
+
+To view information about how MongoDB executes your operation, you can
+instruct the MongoDB query planner to **explain** it. When MongoDB explains
+an operation, it returns **execution plans** and performance statistics.
+An execution plan is a potential way MongoDB can complete an operation.
+When you instruct MongoDB to explain an operation, it returns both the
+plan MongoDB executed and any rejected execution plans.
+
+To explain an aggregation operation, run the ``explain`` database command by passing
+the command information to the ``MongoDB\Database::command()`` method. You must set the
+``aggregate``, ``pipeline``, and ``cursor`` fields of the ``explain`` command document
+to explain the aggregation.
+
+The following example instructs MongoDB to explain the aggregation operation from the
+preceding :ref:`php-aggregation-example`:
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/aggregation.php
+      :start-after: start-explain
+      :end-before: end-explain
+      :language: php
+      :dedent:
+
+   .. output:: 
+      :visible: false
+
+      {"explainVersion":"2","queryPlanner":{"namespace":"sample_restaurants.restaurants",
+      "indexFilterSet":false,"parsedQuery":{"cuisine":{"$eq":"Bakery"}},"queryHash":"865F14C3",
+      "planCacheKey":"D56D6F10","optimizedPipeline":true,"maxIndexedOrSolutionsReached":false,
+      "maxIndexedAndSolutionsReached":false,"maxScansToExplodeReached":false,"winningPlan":{
+      ... }
+
+
+Additional Information
+----------------------
+
+MongoDB Server Manual
+~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I think this would work better as a short sentence followed by a bulleted list for the sentences below. Something like:

To learn more about the topics discussed in this guide, see the following pages in the {+mdb-server+} manual:
- To view a full list of expression operators, see :manual:`Aggregation Operators </reference/operator/aggregation/>`
...
...
...